modules/rosa-view-cloudwatch-logs.adoc

```diff
@@ -0,0 +1,22 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// rosa_logging/rosa-install-logging.adoc
+
+[id="rosa-view-cloudwatch-logs_{context}"]
+= Viewing forwarded logs
+
+Logs that are being forwarded from {product-title} are viewed in the Amazon Web Services (AWS) console.
+
+.Prerequisites
+
+* The `cluster-logging-operator` add-on service is installed.
+* The Amazon CloudWatch add-on service is installed.
+
+.Procedure
+
+. Log in to the AWS console.
+. Select the region the cluster is deployed in.
+. Select the *CloudWatch* service.
+. Select *Logs* from the left column, and select *Log Groups*.
+. Select a log group to explore. You can view application, infrastructure, or audit logs, depending on which types were enabled during the add-on service installation. See the link:https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/WhatIsCloudWatch.html[Amazon CloudWatch User Guide] for more information.
```

Comment:
throw in a screenshot perhaps?

Reply:
It's our policy not to include images from products that we don't own because it's very difficult to track changes UI to their products. To give our customers the best experience, we provide links so that they have direct access to the latest accurate information that our third-party vendor/partner can provide.